src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="mac photo"
+        class="card__photo"
+      />
+      <h2 class="card__tittle">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rate">
+        <div class="card__rate-stars">
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
```

Comment:
The stars in the rating should be created using CSS, not with div elements. You should have a div with a class for the rating and then use CSS to display the stars.